packages/amplication-server/src/core/prismaSchemaUtils/schema-utils.ts

```diff
@@ -206,3 +209,25 @@ export function jsonField(field: Field) {
     return EnumDataType.Json;
   }
 }
+
+export function findFkFieldNameOnAnnotatedField(field: Field): string {
+  const relationAttribute = field.attributes?.find(
+    (attr) => attr.name === "relation"
+  );
+
+  const fieldsArgs = relationAttribute.args.find(
```

Comment:
Please add verification that `relationAttribute` and later on `fieldsArgs` and `fieldsArgsValues` are not null/undefined